config/webpack.config.js

```diff
@@ -236,9 +241,20 @@ module.exports = env => {
       }),
 
       new MiniCssExtractPlugin({
-        filename: !isOptimizedBuild
-          ? '[name].css'
-          : `[name].[contenthash]-${timestamp}.css`,
+        moduleFilename: chunk => {
+          const { name } = chunk;
+          const standardFilename = `${name}.[contenthash]-${timestamp}.css`;
+          const isMedalliaStyleFile = name === vaMedaliaStylesFilename;
+          const isStandardFile = !isMedalliaStyleFile;
+          const showHash = isOptimizedBuild && isStandardFile;
+
+          if (showHash) return standardFilename;
+
+          const isStaging =
+            buildOptions.buildtype === ENVIRONMENTS.VAGOVSTAGING;
+
+          return isStaging ? `${name}.css` : standardFilename;
```

Comment:
Is this for testing on staging first?

Reply:
That's correct. 

>once we can publish the file to staging, we need to give it to insights team to add to the medallia iframe settings. - https://github.com/department-of-veterans-affairs/vets-website/pull/14312#issuecomment-700067751

Reply:
>I think we should have the staging check before any return statements. otherwise this logic is fine.

@rianfowler, I don't have an objection to moving the staging check before any `return`s. That being said, is the Medallia CSS file the only file that cares about the staging check? If so, what is the value of performing the staging check for every file?

Reply:
>Now that I'm getting a second look at this, it seems like dev would keep the hash in the name (not optimized build, so showHash is falsy). Not sure how much this matters.

@U-DON, my intent for the code was:
1. Determine if it's the Medallia CSS file.
1. If not, return our normal filename format.
1. If so, determine if it's the staging* environment.
   * My understanding of the requirements is that I should be checking only for staging. If I should be checking for additional non-prod environments, then I need to update my code. 
1. If staging, exclude the hash. 
1. If non-staging, include the hash.